uui-core/src/types/tables.ts

```diff
@@ -164,6 +166,8 @@ export interface DataTableHeaderRowProps<TItem = any, TId = any> extends IEditab
     onConfigButtonClick?: (params: DataTableConfigModalParams) => any;
     renderCell?: (props: DataTableHeaderCellProps<TItem, TId>) => React.ReactNode;
     renderConfigButton?: () => React.ReactNode;
+    /** Defines table columns gap size */
+    columnsGap?: '12' | '24';
```

Comment:
Why do we need columnsGap in column configuration modal?

Reply:
moved it to the uui/ DataTableHeaderRowMods